tox.ini

```diff
@@ -5,17 +5,18 @@ skipsdist = true
 
 
 [flake8]
-exclude = ownca/__init__.py,venv,.venv,settings.py,.git,.tox,dist,docs,*lib/python*,*egg,build,tools
+exclude = repository_service_tuf/__init__.py,venv,.venv,settings.py,.git,.tox,dist,docs,*lib/python*,*egg,build,tools
 
 [testenv]
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/requirements-dev.txt
 
 [testenv:lint]
+deps = pre-commit
 commands =
-    flake8
-    isort -l79 --profile black --check --diff .
-    black -l79 --check --diff .
+    pre-commit run flake8 --all-files --show-diff-on-failure
+    pre-commit run isort --all-files --show-diff-on-failure
+    pre-commit run black --all-files --show-diff-on-failure
```

Comment:
Thanks for the feedback @MVrachev. 

> What's the difference between `pre-commit run flake8 --all-files --show-diff-on-failure` and `flake8` command?
> 
> I am not sure I understand the benefit of having `pre-commit` here instead of directly calling the linters.

This way you have the same exact commands run in the CI as you have locally by the pre-commit hook (run by the developer).

Another positive side-effect is that you can lint, if you choose it to ([it's the default option](https://pre-commit.com/#4-optional-run-against-all-the-files)), only on the changed files.
So if for instance, you are introducing a new tool, such as pylint, and you want to have a gradual migration and not have edits to almost all your project files to conform to the new style, you only need to apply the changes to the files that have been changed on your last commit.